plugin/hosts/hostsfile.go

```diff
@@ -32,6 +32,26 @@ func absDomainName(b string) string {
 	return plugin.Name(b).Normalize()
 }
 
+type hostsOptions struct {
```

Comment:
just 'options' ?

Reply:
sure, I tried to match the name of HostsMap ..

Reply:
done, however in `setup` we now have a variable with the same name as the struct shadowing it.